README.md

```diff
@@ -11,7 +11,8 @@ Usage
 -----
 
 Add `rspec-rerun` to `Gemfile` in the `:development` and `:test` groups.
-If you're using RSpec 3, also add `rspec-legacy_formatters`.
+
+(For Rspec 2 use version '~> 0.3.1'. )
```

Comment:
It's RSpec with a capital S :) 

I would say "If you're using RSpec 3.x, add ..." and then continue "For RSpec 2.x use version '~> 0.3.1' of rspec-rerun" or something like that. Totally nitpicking feel free to leave it alone.